code/datums/wires/airlock.dm

```diff
@@ -115,7 +115,6 @@ var/const/AIRLOCK_WIRE_LIGHT = 2048
 				//Cutting this wire electrifies the door, so that the next person to touch the door without insulated gloves gets electrocuted.
 				if(A.secondsElectrified != -1)
 					A.shockedby += text("\[[time_stamp()]\][usr](ckey:[usr.ckey])")
-					add_logs(usr, A, "electrified", admin=0, addition="at [A.x],[A.y],[A.z]")
```

Comment:
I'm logging every wire cut, see:
[02:28:17]ATTACK: 9600bauds/(Eli Stamos) has mended the Electrify wire of airlock named "Mechanics Workshop" at 228,220,1.